### PR TITLE
Support and test with Node 8

### PR DESCRIPTION
**Pull request by jeffjewiss**
I updated the supported Node engines to be inline with what is supported by `ember-cli@2.14`. I also updated the Travis config so the same versions of Node should be tested.

Cheers

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,6 +16,7 @@ env:
   - EMBER_TRY_SCENARIO=ember-1.13
   - EMBER_TRY_SCENARIO=ember-lts-2.4
   - EMBER_TRY_SCENARIO=ember-lts-2.8
+  - EMBER_TRY_SCENARIO=ember-lts-2.12
```
Comment (simonihmig) [inline .travis.yml]: This scenario is not defined yet in https://github.com/kaliber5/ember-responsive-image/blob/master/config/ember-try.js

File: package.json [hunk 2]
```diff
@@ -59,7 +59,6 @@
     "ember-fastboot-addon-tests": "0.4.0",
     "ember-load-initializers": "^1.0.0",
     "ember-resolver": "^4.0.0",
-    "ember-source": "~2.13.0",
```
Comment (simonihmig) [inline package.json]: Why did you remove ember? One cannot work in development without it!?

File: .travis.yml [hunk 3]
```diff
@@ -1,7 +1,10 @@
 ---
 language: node_js
 node_js:
+  - "4"
```
Comment (simonihmig) [inline .travis.yml]: Node 4 is failing, seems some missing `'use strict';`. Can you add those?

File: .travis.yml [hunk 4]
```diff
@@ -1,7 +1,10 @@
...
   - "6"
+  - "7"
+  - "8"
```
Comment (simonihmig) [inline .travis.yml]: This setup makes the test suite really huge (and slow!). This gives us numberOfNodeVersions * numberOfEmbyTryScenarios test jobs. Do we really need this, or would testing with node 4 not be enough? 

A setup were we test the different node versions only for one ember-try scenario, and run all scenarios with e.g. node 6 would be nice I think, but no idea how to do this in Travis...

File: package.json (hunk 2 above)
Comment (jeffjewiss) [reply package.json]: Sorry, this was an oversight on my part. I think I may have had a cancelled run of `ember-try` that had some side effects. I'll revert this.

File: .travis.yml (hunk 1 above)
Comment (jeffjewiss) [reply .travis.yml]: My bad, I'll add this scenario to ember-try.

File: .travis.yml (hunk 3 above)
Comment (jeffjewiss) [reply .travis.yml]: Yeah! I'll make sure to test locally against Node 4 and fix those.
